libyul/backends/wasm/WordSizeTransform.cpp

```diff
@@ -268,15 +271,17 @@ void WordSizeTransform::rewriteIdentifierList(vector<Identifier>& _ids)
 	);
 }
 
-void WordSizeTransform::rewriteFunctionCallArguments(vector<Expression>& _args)
+void WordSizeTransform::rewriteFunctionCallArguments(vector<Expression>& _args, vector<bool> const* _literalArgs)
```

Comment:
I think it is better to "inline" this function - it is only called once and passing `_literalArgs` by pointer does not make it clearer.